### DISCUSSION TITLE
Shady Conclusions; Steady Misuses

### DISCUSSION THREAD

**Discussion by brillianz** [albertkun/21S-ASIAAM-191A#45] [Past Thinking Caps]
To start, data has historically been considered through the guise of objectivity. Statistics, big data, and research all revolves its motives and approaches through a particular worldview. To this end, these worldviews are informed by and inform different agendas. If we assume that there are no agendas associated with data or motives, then we run the risk of being nonchalant towards other, more dominant agendas. 
An extension of this discussion finds itself within the visualizations produced by GIS. These numbers can be considered and influenced by the authors of the visualizations, and, moreover, can be manipulated according to the scale, proportions, and severity of factors. This is especially the case for countries in the global south, who are often less so the authors of these visualizations and instead those that are interpreted. 

][

**Comment by lvalles10** [comment]
To your last point, the De Young in San Francisco is currently hosting an exhibit on the weaponization of AI to oppress the global south among other relevant topics on data (https://deyoung.famsf.org/exhibitions/uncanny-valley). It's terrifying to see the disastrous effects AI and the manipulation of data can have on the identities of marginalized and oppressed individuals in these countries. Various artists in the exhibit acknowledge how the different worldviews of various influential players in the data game (i.e. Big Tech, governments, resistance groups, and other nefarious actors) can impact broader citizenry.
